nss-day-cohort-43-csharp/Fullstack-Rage-Against-The-Virtual-Machine
# Bl list tags

# Description
Admins can now navigate to a list of tags
Fixes # 16
## Type of change
Please delete options that are not relevant.
# Testing Instructions
- Login as an admin
- Click the Tags tab to make sure a list of tags are displayed
- Logout and log back in as an author
- Navigate to /tags and make sure it navigates to the home page
- Logout and do not log back in
- Navigate to /tabs again and ensure that it navigates to the login page
- On Postman, try to get the tags without a token
- On Postman, try to get the tags with the token of an author
# Checklist:
